app/code/Magento/InventoryShipping/README.md

```diff
@@ -1 +1,24 @@
-# InventoryShipping
+# InventoryShipping module
+
+The `InventoryShipping` module integrates MSI business logic into Magento's shipping logic.
+
+This module is part of the MSI (Multi-Source Inventory) project. The 
+[Inventory Management overview](https://devdocs.magento.com/guides/v2.3/inventory/index.html)
+describes the MSI project in more detail.
+
+## Installation details
+ 
+This module is installed as part of Magento Open Source. It cannot be deleted or disabled.
+
+## Extension points and service contracts
+
+All public interfaces related to this module are located in the `InventorySourceDeductionApi` and 
+`InventorySourceSelectionApi` modules. 
+Use the interfaces defined in those modules to extend this module.
+
+## Additional information
+
+The `InventoryShipping` module defines the following events:
```

Comment:
it does not define events, it provides listeners/handlers for these events.
Consumers are not APIs